core/templates/pages/release-coordinator-page/features-tab/features-tab.component.html

```diff
@@ -245,4 +246,11 @@ <h2 class="oppia-feature-name e2e-test-feature-name">{{featureFlag.name}}</h2>
   .e2e-test-angular-dummy-feature-indicator {
     bottom: 10px;
   }
+
+  @media screen and (max-width: 767px) {
+    .col-2 {
```

Comment:
I think col-2 is an mdbootstrap class. Ideally, we should not be overriding classes that belong to third-party libraries as it can lead to unexpected issues. Can we provide styling to a custom defined class instead?

Reply:
Done, added a custom class